app/src/androidTest/java/com/sdp/movemeet/view/map/GPSRecordingActivityTest.java

```diff
@@ -1,11 +1,13 @@
 package com.sdp.movemeet.view.map;
 
 import android.Manifest;
+import android.content.Intent;
```

Comment:
There are a lot of import, have you deleted the non-useful ones ?

Reply:
Admittedly, I don't think I cleaned up the imports before committing. Thank you for pointing it out!